Dockerfile

```diff
@@ -1,13 +1,42 @@
-FROM ubuntu:19.04
-RUN apt update && apt install -y wget ca-certificates
+FROM node:10.16.3-jessie as build-env
 
-ARG DRONE_TAG
+### Install Go ###
+ENV GO_VERSION=1.12.10 \
+    GOPATH=$HOME/go-packages \
+    GOROOT=$HOME/go
+ENV PATH=$GOROOT/bin:$GOPATH/bin:$PATH
+RUN curl -fsSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.tar.gz | tar -xzv \
+    && go get -u -v \
+        github.com/acroca/go-symbols \
+        github.com/cweill/gotests/... \
+        github.com/davidrjenni/reftools/cmd/fillstruct \
+        github.com/fatih/gomodifytags \
+        github.com/haya14busa/goplay/cmd/goplay \
+        github.com/josharian/impl \
+        github.com/nsf/gocode \
+        github.com/ramya-rao-a/go-outline \
+        github.com/rogpeppe/godef \
+        github.com/uudashr/gopkgs/cmd/gopkgs \
+        github.com/zmb3/gogetdoc \
+        golang.org/x/lint/golint \
+        golang.org/x/tools/cmd/godoc \
+        golang.org/x/tools/cmd/gorename \
+        golang.org/x/tools/cmd/guru \
+        sourcegraph.com/sqs/goreturns \
+        github.com/UnnoTed/fileb0x
 
-RUN wget -O /tmp/xbvr.tgz "https://github.com/xbapps/xbvr/releases/download/"$DRONE_TAG"/xbvr_"$DRONE_TAG"_Linux_x86_64.tar.gz" && \
-    tar xvfz /tmp/xbvr.tgz -C /usr/local/bin/ && \
-    rm /tmp/xbvr.tgz
+WORKDIR /app
+ADD . /app
+RUN cd /app && \
+    yarn install && \
+    yarn build && \
+    go generate && \
+    go build -tags='json1' -ldflags '-w' -o xbvr main.go
+
+FROM FROM gcr.io/distroless/base-debian10
```

Comment:
`FROM FROM`? 😄 

Also, why debian10?

Reply:
grrr... I fixed that and then something must have reverted. I'll fix it.

Reply:
Google updated the distroless to debian10 but they pulled the images for some reason so I reverted it back to base.